ios/PurchasesHybridCommon/PurchasesHybridCommon/Constants.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Constants.swift
+//  PurchasesHybridCommon
+//
+//  Created by Jay Shortway on 19/06/2024.
+//  Copyright © 2024 RevenueCat. All rights reserved.
+//
+
+import Foundation
+
+public struct Constants {
```

Comment:
Hmm I think we shouldn't make this public, but `internal` and just expose it through the `CommonFunctionality` class right?

Reply:
Very much so yes! Good catch! https://github.com/RevenueCat/purchases-hybrid-common/pull/838/commits/5e44a5751f4c3ed47fac548fc1853ae7274d7ec1